Tensile/Common.py

```diff
@@ -136,13 +136,18 @@
 # trig_float initializes with the sin function to have non-zero values in the mantissa
 # and exponent. It cannot be used for int8 or int32. Need to use tensileAlmostEqual
 # not tensileEqual for checking the result.
+# NOTE- the following comments explaining the DataInitType are for OldClient and should be obsolete.
```

Comment:
I think maybe the comments for alpha/beta type still holds correct?

Reply:
alpha / beta , the old comments (0~4) holds correct for new client as well.
But in new client, you can also set Alpha/Beta type to 5,6,7,16 (Inf, BadIn, BadOut, rand-narrow) and it works in host
BTW, if you set Alpha/Beta to 8~15, (problem-dept.), client writer doesn't show any warning,
but host-client will send a "Invalid InitMode" exception.

Reply:
We may simply make clear the acceptable values for A/B/C/D & Alpha/Beta from end user's perspective

Reply:
Updated